ru/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -1,9 +1,9 @@
 <!ENTITY pbMacroSearchPanel.width "600">
 
 <!ENTITY insertMacro.key "l">
-<!ENTITY insertMacro.modifiers "accel">
+<!ENTITY insertMacro.modifiers "����������">
 
-<!ENTITY insertMacro.label "Quick Bar">
+<!ENTITY insertMacro.label "������ �������� �������">
```

Comment:
This is outdated file, it was in the ANSI encoding and therefore you see the garbage characters. The correct encoding is UTF-8 and I have fixed it on Aug 22nd. Please use the latest corrected file.